users/views.py

```diff
@@ -1,12 +1,17 @@
-from django.shortcuts import render
 import logging
+from rest_framework_simplejwt.views import TokenObtainPairView
+from .serializers import CustomTokenObtainPairSerializer
 
-logger = logging.getLogger(__name__)
+# Custom view to use the custom JWT serializer
+class CustomTokenObtainPairView(TokenObtainPairView):
+    serializer_class = CustomTokenObtainPairSerializer
 
+# Optional logging setup (can be removed if not needed)
+logger = logging.getLogger(__name__)
 logger.debug("This is a debug message.")
 logger.info("Informational message.")
 logger.warning("Warning occurred!")
 logger.error("An error happened.")
 logger.critical("Critical issue!")
```

Comment:
Are these logger calls meant for debugging during development?
If so, consider removing or commenting them out before merging, as they will generate log output on every import of this module. If you want to keep them, consider wrapping them in a `name == "main"` block or moving them into actual view logic.